srcpkgs/pulseeffects-legacy/template

```diff
@@ -7,15 +7,14 @@ build_style=meson
 hostmakedepends="itstool pkg-config gettext"
 makedepends="boost-devel glib-devel gsettings-desktop-schemas-devel
  gst-plugins-bad1-devel gtkmm-devel libebur128-devel lilv-devel
- pulseaudio-devel python3-gobject-devel sratom-devel
- libsndfile-devel"
-depends="calf gsettings-desktop-schemas gst-plugins-bad1
- gst-plugins-good1 pulseaudio python3-gobject python3-scipy"
+ pulseaudio-devel sratom-devel zita-convolver-devel
+ libsamplerate-devel libsndfile-devel"
+depends="calf gst-plugins-good1"
 short_desc="Sound effects for systems using PulseAudio (legacy)"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/wwmm/pulseeffects"
+changelog="https://github.com/wwmm/pulseeffects/blob/master/CHANGELOG.md"
```

Comment:
changelog is usually a link to a raw text file: `https://raw.githubusercontent.com/wwmm/pulseeffects/master/CHANGELOG.md`

Reply:
@paper42 https://github.com/void-linux/void-packages/pull/31667#discussion_r659343029
> I want to click on the link in my terminal
Why would you want to open raw text instead of pretty rendered Markdown?

Reply:
I don't care about how pretty a changelog is, please follow the convention.

Reply:
Yet Another Unwritten Void Distro Policy
OK, 'll change it when reach to the laptop later in the day.

Reply:
@paper42 fixed